.github/workflows/ci.yaml

```diff
@@ -315,10 +315,46 @@ jobs:
           path: ./logs/
           retention-days: 15
 
-  push-gpu-operator-bundle-image:
+  gpu-operator-and-validator-image-retag:
```

Comment:
Can we break this action stage down two? One for gpu-operator and another for gpu-operator-validator. Let's rename this to something like 

```
release-staging-gpu-operator-image
release-staging-gpu-operator-validator-image
```

Reply:
OR
```
release-latest-gpu-operator-image
release-latest-gpu-operator-validator-image
```